skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
+        # utility for maintaining previous color image default behavior
+        if ndim == 3:
+            return True
+        else:
+            return False
+
+
 def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
            preserve_range=False):
```

Comment:
@soupault but the API is necessarily different — rescale can take float input, while resize *always* needs a tuple. This means that the multichannel nature can be inferred from the output and input shapes. At any rate, I consider this something that can be fixed post merge if necessary.